### PR TITLE
update extension for gnome-shell 3.30

### PR DESCRIPTION
**Pull request by xmsk**
I have added some changes such that the extension works in gnome-shell 3.30. I don't know if it will work in older versions, though.

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ function goToLastWorkspace() {
     return;
   }
 
-  let ws = global.screen.get_workspace_by_index(lastWorkspace);
```
Comment (arjan) [inline extension.js]: Can you try to keep it BC?

It wouldn't be too hard to keep backward compatibility by using the construct `(global.workspace_manager || global.screen).get…`, right?

File: extension.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ function enable() {
   var ModeType = Shell.hasOwnProperty('ActionMode') ? Shell.ActionMode : Shell.KeyBindingMode;
   Main.wm.addKeybinding(SHORTCUT_KEY, settings, Meta.KeyBindingFlags.NONE, ModeType.NORMAL | ModeType.OVERVIEW, goToLastWorkspace);
 
-  signals.push(global.screen.connect('workspace-switched', function(screen, prev, current, direction) {
+  signals.push(global.workspace_manager.connect('workspace-switched', function(display, prev, current, direction) {
```
Comment (arjan) [inline extension.js]: here as well

**Comment by xmsk** [comment]
I have added your suggestions and tested it on 3.28. Do you have any other suggestions?

File: extension.js (hunk 1 above)
Comment (xmsk) [reply extension.js]: Added your suggestions. It works in 3.28 as well now.

**Comment by arjan** [comment]
LGTM, thanks!

**Pull request merged**